packages/types/types/index.d.ts

```diff
@@ -193,18 +194,73 @@ export interface ThemeProviderProps {
   customBreakpoints?: string[]
 }
 
+export interface DividerProps
+  extends ColorProps,
+    SpaceProps,
+    WidthProps,
+    BorderColorProps {}
+
+export interface AvatarProps extends ClassNameProps, TextColorProps {
+  /** Heading displayed with avatar */
+  title?: string,
+  /** Subheading displayed beneath heading */
+  subtitle?: string,
+  src?: string,
+  /** Alternative initials to display when src is not provided */
+  initials?: string,
+  /** Size in px to use for height, width, and background-size */
+  size?: number
+}
+
+export interface BannerProps extends BoxProps {
+  header?: string | React.ComponentElement<IconProps, any>,
+  icon?: React.ComponentElement<IconProps, any>,
+  onClose?: (any) => void,
+  showIcon?: boolean,
+  text?: string | React.ReactNode
+}
+
+export interface StampProps
+  extends SpaceProps, FontSizeProps, ColorProps {
+    variation?: 'outline' | 'fill' | 'solid',
+    size?: 'small' | 'medium'
+  }
+
+export interface ContainerProps {
+  maxWidth?: number
+}
+
+export interface FlagProps extends FlexProps,
+    ColorProps,
+    WidthProps {}
+
+export interface HugProps extends CardProps, ColorProps {
+  iconDisplay?: string[],
+  icon?: React.ComponentElement<IconProps, any>,
+  text?: React.ReactNode | React.ReactNode[] | string
```

Comment:
I think this can be changed to `  text?: ReactNode | string` because `ReactNode` already handles an array of nodes:

```
    interface ReactNodeArray extends Array<ReactNode> {}
    type ReactFragment = {} | ReactNodeArray;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```

Reply:
Good to know for future reference, thank you!